<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPlayer</class>
 <widget class="QWidget" name="AddPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>123</x>
     <y>90</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Positon</string>
   </property>
  </widget>
  <widget class="QComboBox" name="positionComboBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>CF</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RW</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LW</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CAM</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CM</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CDM</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GK</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="shirtLineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="appLineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="goalLineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="assistLineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="passLineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="tackleLineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="dribbleLineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>300</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="yellowLineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>330</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="redLineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>360</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="savePer90LineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>390</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="cleanSheet">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>420</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="saveRateLineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>450</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>490</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>93</x>
     <y>50</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Shirt Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>123</x>
     <y>120</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Appearances</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Goals</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>123</x>
     <y>210</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Assists</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Pass completed per 90</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Tackle completed per 90</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Dribble completed per 90</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>330</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Yellow Card</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>360</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Red Card</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Save performed per 90</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of clean sheet</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Save success percentage</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="saveRateLineEdit_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>560</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmButton_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>600</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>560</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Save success percentage</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
